tests/libcxx/enc/CMakeLists.txt

```diff
@@ -82,7 +82,8 @@ foreach(testcase ${alltests})
 # the allocations are entirely optimized out by Clang in these tests and are excluded from Clang release builds
     if ("${name}" MATCHES "cons_default_throws_bad_alloc.pass" OR "${name}" MATCHES "allocator_allocator.members_construct.pass")
         string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UPPER)
-        if (CMAKE_CXX_COMPILER_ID MATCHES Clang AND BUILD_TYPE_UPPER MATCHES REL)
+        if ((CMAKE_CXX_COMPILER_ID MATCHES Clang OR ENCLAVE_TESTS_CXX_COMPILER_ID MATCHES Clang)
```

Comment:
This file shouldn't require any changes since the enclaves are only built on Linux right now anyway, no? (`ENCLAVE_TESTS_CXX_COMPILER_ID` doesn't get set if `BUILD_ENCLAVES` is set, and `BUILD_ENCLAVES` is required to build this folder.)

Reply:
You are correct, undo the modifications.